functionary/ui/templates/partials/task_result_block.html

```diff
@@ -1,22 +1,28 @@
-{% if completed %}
-    <div id="result-block" class="block ml-4 is-12" hx-swap-oob="true">
-        <div class="tabs is-small is-toggle">
-            <ul hx-target="#result-block">
-                <li {% if output_format != "table" and output_format != "log" %}class="is-active"{% endif %}>
-                    <a hx-get="{% url 'ui:task-results' task.id %}?output=display_raw">Raw</a>
-                </li>
-                {% if show_output_selector %}
-                    <li {% if output_format == "table" %}class="is-active"{% endif %}>
-                        <a hx-get="{% url 'ui:task-results' task.id %}?output=display_table">Table</a>
-                    </li>
-                {% endif %}
-                <li {% if output_format == "log" %}class="is-active"{% endif %}>
-                    <a hx-get="{% url 'ui:task-log' pk=task.id %}">Log</a>
-                </li>
-            </ul>
-        </div>
-        {% include "partials/task_result.html" %}
+<div id="result-block" hx-swap-oob="true">
+    <div class="ps-4">
+        {% if completed %}
+            <div class="py-3" hx-target="#result-block">
+                <div aria-label="Result output type">
+                    <ul class="pagination">
+                        <li class="page-item {% if output_format != "table" and output_format != "log" %}active{% endif %}">
+                            <a hx-get="{% url 'ui:task-results' task.id %}?output=display_raw"
```

Comment:
There's a slightly weird behavior here where the cursor doesn't update to a pointer like you'd expect.  This seems to be because we have an <a> with no `href`, so the browser pointer for links doesn't get applied.  And none of the classes we're using explicitly change the pointer.  Two options come to mind and I'm sure there are more:

* Make these <button> rather than <a>.  That seems to maintain the look, but get us the expected cursor.  It may be more accurate too since I guess they are behaving more like buttons than links.
* Add an `href="#"` or something similar so that the default browser styling is applied.

I don't know which of those seems more appropriate.  There may be other options too.